mantle/kola/tests/misc/chronydhcp.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package misc
+
+import (
+	"strings"
+	"time"
+
+	"github.com/coreos/mantle/kola"
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+	"github.com/coreos/mantle/util"
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Name:        "coreos.chrony.dhcp.verify",
+		Run:         verifyDHCPPropagationOfNTPServers,
+		ClusterSize: 1,
+		Tags:        []string{"chrony", "dhcp", "ntp", kola.NeedsInternetTag},
+		UserDataV3: conf.Ignition(`{
+			"ignition": {"version": "3.0.0"},
```

Comment:
Should we use the latest version supported by both RHCOS and FCOS?

```suggestion
			"ignition": {"version": "3.1.0"},
```

Reply:
Ignition automatically uptranslates all configs to the latest version internally.

Reply:
So it doesn't matter what we do here?

Though if you were going to create a new test, what would you do?

Reply:
> So it doesn't matter what we do here?

Correct, if it did that'd be a bug in Ignition.

> Though if you were going to create a new test, what would you do?

I personally would probably go for the earliest version inside the major spec that has all of the required fields.